src/Components/AddProduct/AddProduct.tsx

```diff
@@ -11,27 +12,64 @@ import { productFormList } from '../../Constants/branch';
 import { AddAPhoto } from '@mui/icons-material';
 import { theme } from '../../Constants/theme';
 import './AddProduct.scss'
+
+
 function AddPicture() {
+    const customInput=useRef<null | HTMLInputElement>(null);
+    const [viewImage,setViewImage]=React.useState(false);
+    const [file,setFile]=React.useState<File>();
+    function getFile(event:React.BaseSyntheticEvent){
+        console.log(event.target.files[0])
+        setFile(event.target.files[0])
+    }
+    function handleClick(){
+        if(customInput.current!=null){
```

Comment:
follow proper spacing customInput != null

Reply:
Thanks for suggestion